### PR TITLE
Update Web/API/Request

### PR DESCRIPTION
**Pull request by mfuji09**
> What was wrong/why is this fix needed? (quick summary only)

Request.bodyUsed actually returns true or false primitive, not a Boolean wrapper object.

> Issue number (if there is an associated issue)

None.

### CODE REVIEW

**Comment by peterbe** [comment]
@mfuji09 I think you might need to merge in the latest `main` into this branch to make it work again.

File: files/en-us/web/api/request/index.html [hunk 1]
```diff
@@ -30,7 +30,7 @@ <h2 id="Properties">Properties</h2>
  <dt>{{domxref("Request.body")}} {{readonlyInline}}</dt>
  <dd>A {{domxref("ReadableStream")}} of the body contents.</dd>
  <dt>{{domxref("Request.bodyUsed")}} {{readonlyInline}}</dt>
- <dd>Stores a {{domxref("Boolean")}} that declares whether the body has been used in a request yet.</dd>
+ <dd>Stores <code>true</code> or <code>false</code> that declares whether the body has been used in a request yet.</dd>
```
Comment (wbamberg) [inline files/en-us/web/api/request/index.html:33]:
```suggestion
 <dd>Stores <code>true</code> or <code>false</code> to indicate whether or not the body has been used in a request yet.</dd>
```

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thanks @mfuji09 ! I had a copy-editing suggestion, but otherwise this makes sense to me.

File: files/en-us/web/api/request/index.html (hunk 1 above)
Comment (mfuji09) [reply files/en-us/web/api/request/index.html]: Thank you! It looks much better!

**Review by wbamberg — APPROVED** [review]
👍 thanks!

**Pull request merged**